app/api/dao/mentorship_relation.py

```diff
@@ -183,6 +183,10 @@ def accept_request(user_id, request_id):
             if request.state == MentorshipRelationState.ACCEPTED:
                 return messages.USER_IS_INVOLVED_IN_A_MENTORSHIP_RELATION, 400
 
+        # verify if user is involved in other mentorship relation
```

Comment:
This is not required `USER_IS_INVOLVED_IN_A_MENTORSHIP_RELATION` is already there

Reply:
Updated!